docker-compose.yaml

```diff
@@ -116,21 +140,22 @@ services:
       - "7001:7000"
 
   yb-tserver-0:
-    image: yugabytedb/yugabyte:2.15.3.0-b231
+    <<: *common
+    image: *yb-image
     container_name: yb-tserver-0
     hostname: yb-tserver-0
     volumes:
       - ./products_service/:/app
-    command: bash -c "
-              rm -rf /tmp/.yb* ; 
-              /home/yugabyte/bin/yb-tserver --ysql_beta_feature_tablespace_alteration=true --ysql_enable_packed_row=true --ysql_beta_features=true --yb_enable_read_committed_isolation=true --default_memory_limit_to_ram_ratio=0.20
-              --placement_cloud=cloud 
-              --placement_region=region 
-              --placement_zone=zone 
-              --enable_ysql=true 
-              --fs_data_dirs=/home/yugabyte/data 
-              --rpc_bind_addresses=yb-tserver-0:9100 
-              --tserver_master_addrs=yb-master-0:7100,yb-master-1:7100,yb-master-2:7100 
+    command: &tserver-command bash -c "
+              rm -rf /tmp/.yb* ;
+              exec /home/yugabyte/bin/yb-tserver --ysql_beta_feature_tablespace_alteration=true --ysql_enable_packed_row=true --ysql_beta_features=true --yb_enable_read_committed_isolation=true --default_memory_limit_to_ram_ratio=0.20
+              --placement_cloud=cloud
+              --placement_region=region
+              --placement_zone=zone
```

Comment:
Not sure if these `--placement_*` options are needed. They seem to be used inconsistently before this PR.